trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+class TrimDecorator < Decorator
+  def correct_name
+    max_length = 10
```

Comment:
- [ ] kindly remember that it is strings are index-based and it starts from 0 so if you include 10, then it makes it 11, I would implore you to make it 9 so that it would be 10 - meaning 0-9.
- [ ] Kindly remember to require_relative './decorator'